tests/mpmc_channel.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(async_await)]
+#![feature(async_closure)]
```

Comment:
Interesting. I didn't actually thought one of those had been used. Should probably be cleaned up.